[
  {
    "cveID": "CVE-2019-17195",
    "cveASSIGNER": "cve@mitre.org",
    "publishedDate": "2019-10-15T14:15Z",
    "lastModifiedDate": "2021-05-07T11:15Z",
    "description": [
      {
        "lang": "en",
        "value": "Connect2id Nimbus JOSE+JWT before v7.9 can throw various uncaught exceptions while parsing a JWT, which could result in an application crash (potential information disclosure) or a potential authentication bypass."
      }
    ],
    "references": [
      {
        "url": "https://connect2id.com/blog/nimbus-jose-jwt-7-9",
        "name": "https://connect2id.com/blog/nimbus-jose-jwt-7-9",
        "refsource": "CONFIRM",
        "tags": [
          "Release Notes",
          "Vendor Advisory"
        ]
      },
      {
        "url": "https://bitbucket.org/connect2id/nimbus-jose-jwt/src/master/SECURITY-CHANGELOG.txt",
        "name": "https://bitbucket.org/connect2id/nimbus-jose-jwt/src/master/SECURITY-CHANGELOG.txt",
        "refsource": "CONFIRM",
        "tags": [
          "Release Notes",
          "Third Party Advisory"
        ]
      },
      {
        "url": "https://lists.apache.org/thread.html/8768553cda5838f59ee3865cac546e824fa740e82d9dc2a7fc44e80d@%3Ccommon-dev.hadoop.apache.org%3E",
        "name": "[hadoop-common-dev] 20191107 [jira] [Created] (HADOOP-16690) Update dependency com.nimbusds:nimbus-jose-jwt due to security vulnerability",
        "refsource": "MLIST",
        "tags": [
          "Mailing List",
          "Third Party Advisory"
        ]
      },
      {
        "url": "https://lists.apache.org/thread.html/e10d43984f39327e443e875adcd4a5049193a7c010e81971908caf41@%3Ccommon-issues.hadoop.apache.org%3E",
        "name": "[hadoop-common-issues] 20191107 [jira] [Created] (HADOOP-16690) Update dependency com.nimbusds:nimbus-jose-jwt due to security vulnerability",
        "refsource": "MLIST",
        "tags": [
          "Mailing List",
          "Third Party Advisory"
        ]
      },
      {
        "url": "https://www.oracle.com/security-alerts/cpuapr2020.html",
        "name": "N/A",
        "refsource": "N/A",
        "tags": [
          "Patch",
          "Third Party Advisory"
        ]
      },
      {
        "url": "https://www.oracle.com/security-alerts/cpujan2021.html",
        "name": "https://www.oracle.com/security-alerts/cpujan2021.html",
        "refsource": "MISC",
        "tags": [
          "Patch",
          "Third Party Advisory"
        ]
      },
      {
        "url": "https://lists.apache.org/thread.html/rcac26c2d4df22341fa6ebbfe93ba1eff77d2dcd3f6106a1dc1f9ac98@%3Cdev.avro.apache.org%3E",
        "name": "[avro-dev] 20210415 [jira] [Created] (AVRO-3111) CVE-2019-17195",
        "refsource": "MLIST",
        "tags": [
          "Mailing List",
          "Third Party Advisory"
        ]
      },
      {
        "url": "https://lists.apache.org/thread.html/r35f6301a3e6a56259224786dd9c2a935ba27ff6b494d15a3b66efe6a@%3Cdev.avro.apache.org%3E",
        "name": "[avro-dev] 20210416 [jira] [Commented] (AVRO-3111) CVE-2019-17195",
        "refsource": "MLIST",
        "tags": [
          "Mailing List",
          "Third Party Advisory"
        ]
      },
      {
        "url": "https://lists.apache.org/thread.html/r33dc233634aedb04fa77db3eb79ea12d15ca4da89fa46a1c585ecb0b@%3Ccommits.druid.apache.org%3E",
        "name": "[druid-commits] 20210506 [GitHub] [druid] jihoonson commented on a change in pull request #11215: Suppressing false positive CVE-2020-7791",
        "refsource": "MLIST",
        "tags": []
      },
      {
        "url": "https://lists.apache.org/thread.html/r2667286c8ceffaf893b16829b9612d8f7c4ee6b30362c6c1b583e3c2@%3Ccommits.druid.apache.org%3E",
        "name": "[druid-commits] 20210506 [GitHub] [druid] maytasm commented on a change in pull request #11215: Suppressing false positive CVE-2020-7791",
        "refsource": "MLIST",
        "tags": []
      },
      {
        "url": "https://lists.apache.org/thread.html/r5e08837e695efd36be73510ce58ec05785dbcea077819d8acc2d990d@%3Ccommits.druid.apache.org%3E",
        "name": "[druid-commits] 20210507 [druid] branch 0.21.1 updated: Suppressing false positive CVE-2020-7791 (#11215) (#11217)",
        "refsource": "MLIST",
        "tags": []
      }
    ]
  },
  {
    "cveID": "CVE-2013-4492",
    "cveASSIGNER": "secalert@redhat.com",
    "publishedDate": "2013-12-07T00:55Z",
    "lastModifiedDate": "2021-05-04T09:08Z",
    "description": [
      {
        "lang": "en",
        "value": "Cross-site scripting (XSS) vulnerability in exceptions.rb in the i18n gem before 0.6.6 for Ruby allows remote attackers to inject arbitrary web script or HTML via a crafted I18n::MissingTranslationData.new call."
      }
    ],
    "references": [
      {
        "url": "http://weblog.rubyonrails.org/2013/12/3/Rails_3_2_16_and_4_0_2_have_been_released/",
        "name": "http://weblog.rubyonrails.org/2013/12/3/Rails_3_2_16_and_4_0_2_have_been_released/",
        "refsource": "CONFIRM",
        "tags": [
          "Patch",
          "Vendor Advisory"
        ]
      },
      {
        "url": "https://github.com/svenfuchs/i18n/commit/92b57b1e4f84adcdcc3a375278f299274be62445",
        "name": "https://github.com/svenfuchs/i18n/commit/92b57b1e4f84adcdcc3a375278f299274be62445",
        "refsource": "CONFIRM",
        "tags": []
      },
      {
        "url": "https://groups.google.com/forum/message/raw?msg=ruby-security-ann/pLrh6DUw998/bLFEyIO4k_EJ",
        "name": "[ruby-security-ann] 20131203 [CVE-2013-4491] Reflective XSS Vulnerability in Ruby on Rails",
        "refsource": "MLIST",
        "tags": []
      },
      {
        "url": "http://lists.opensuse.org/opensuse-updates/2013-12/msg00093.html",
        "name": "openSUSE-SU-2013:1930",
        "refsource": "SUSE",
        "tags": []
      },
      {
        "url": "http://www.debian.org/security/2013/dsa-2830",
        "name": "DSA-2830",
        "refsource": "DEBIAN",
        "tags": []
      },
      {
        "url": "http://www.securityfocus.com/bid/64076",
        "name": "64076",
        "refsource": "BID",
        "tags": []
      }
    ]
  },
  {
    "cveID": "CVE-2021-2271",
    "cveASSIGNER": "secalert_us@oracle.com",
    "publishedDate": "2021-04-22T22:15Z",
    "lastModifiedDate": "2021-04-29T15:16Z",
    "description": [
      {
        "lang": "en",
        "value": "Vulnerability in the Oracle Work in Process product of Oracle E-Business Suite (component: Resource Exceptions). Supported versions that are affected are 12.1.3 and 12.2.3-12.2.8. Easily exploitable vulnerability allows low privileged attacker with network access via HTTP to compromise Oracle Work in Process. Successful attacks of this vulnerability can result in unauthorized creation, deletion or modification access to critical data or all Oracle Work in Process accessible data as well as unauthorized access to critical data or complete access to all Oracle Work in Process accessible data. CVSS 3.1 Base Score 8.1 (Confidentiality and Integrity impacts). CVSS Vector: (CVSS:3.1/AV:N/AC:L/PR:L/UI:N/S:U/C:H/I:H/A:N)."
      }
    ],
    "references": [
      {
        "url": "https://www.oracle.com/security-alerts/cpuapr2021.html",
        "name": "https://www.oracle.com/security-alerts/cpuapr2021.html",
        "refsource": "MISC",
        "tags": [
          "Patch",
          "Vendor Advisory"
        ]
      }
    ]
  },
  {
    "cveID": "CVE-2021-21368",
    "cveASSIGNER": "security-advisories@github.com",
    "publishedDate": "2021-03-12T17:15Z",
    "lastModifiedDate": "2021-03-19T19:35Z",
    "description": [
      {
        "lang": "en",
        "value": "msgpack5 is a msgpack v5 implementation for node.js and the browser. In msgpack5 before versions 3.6.1, 4.5.1, and 5.2.1 there is a \"Prototype Poisoning\" vulnerability. When msgpack5 decodes a map containing a key \"__proto__\", it assigns the decoded value to __proto__. Object.prototype.__proto__ is an accessor property for the receiver's prototype. If the value corresponding to the key __proto__ decodes to an object or null, msgpack5 sets the decoded object's prototype to that value. An attacker who can submit crafted MessagePack data to a service can use this to produce values that appear to be of other types; may have unexpected prototype properties and methods (for example length, numeric properties, and push et al if __proto__'s value decodes to an Array); and/or may throw unexpected exceptions when used (for example if the __proto__ value decodes to a Map or Date). Other unexpected behavior might be produced for other types. There is no effect on the global prototype. This \"prototype poisoning\" is sort of a very limited inversion of a prototype pollution attack. Only the decoded value's prototype is affected, and it can only be set to msgpack5 values (though if the victim makes use of custom codecs, anything could be a msgpack5 value). We have not found a way to escalate this to true prototype pollution (absent other bugs in the consumer's code). This has been fixed in msgpack5 version 3.6.1, 4.5.1, and 5.2.1. See the referenced GitHub Security Advisory for an example and more details."
      }
    ],
    "references": [
      {
        "url": "https://github.com/mcollina/msgpack5/releases/tag/v3.6.1",
        "name": "https://github.com/mcollina/msgpack5/releases/tag/v3.6.1",
        "refsource": "MISC",
        "tags": [
          "Release Notes",
          "Third Party Advisory"
        ]
      },
      {
        "url": "https://github.com/mcollina/msgpack5/releases/tag/v5.2.1",
        "name": "https://github.com/mcollina/msgpack5/releases/tag/v5.2.1",
        "refsource": "MISC",
        "tags": [
          "Release Notes",
          "Third Party Advisory"
        ]
      },
      {
        "url": "https://github.com/mcollina/msgpack5/releases/tag/v4.5.1",
        "name": "https://github.com/mcollina/msgpack5/releases/tag/v4.5.1",
        "refsource": "MISC",
        "tags": [
          "Release Notes",
          "Third Party Advisory"
        ]
      },
      {
        "url": "https://github.com/mcollina/msgpack5/commit/d4e6cb956ae51c8bb2828e71c7c1107c340cf1e8",
        "name": "https://github.com/mcollina/msgpack5/commit/d4e6cb956ae51c8bb2828e71c7c1107c340cf1e8",
        "refsource": "MISC",
        "tags": [
          "Patch",
          "Third Party Advisory"
        ]
      },
      {
        "url": "https://github.com/mcollina/msgpack5/security/advisories/GHSA-gmjw-49p4-pcfm",
        "name": "https://github.com/mcollina/msgpack5/security/advisories/GHSA-gmjw-49p4-pcfm",
        "refsource": "CONFIRM",
        "tags": [
          "Exploit",
          "Third Party Advisory"
        ]
      },
      {
        "url": "https://www.npmjs.com/package/msgpack5",
        "name": "https://www.npmjs.com/package/msgpack5",
        "refsource": "MISC",
        "tags": [
          "Product",
          "Third Party Advisory"
        ]
      }
    ]
  },
  {
    "cveID": "CVE-2021-0211",
    "cveASSIGNER": "sirt@juniper.net",
    "publishedDate": "2021-01-15T18:15Z",
    "lastModifiedDate": "2021-01-22T14:55Z",
    "description": [
      {
        "lang": "en",
        "value": "An improper check for unusual or exceptional conditions in Juniper Networks Junos OS and Junos OS Evolved Routing Protocol Daemon (RPD) service allows an attacker to send a valid BGP FlowSpec message thereby causing an unexpected change in the route advertisements within the BGP FlowSpec domain leading to disruptions in network traffic causing a Denial of Service (DoS) condition. Continued receipt of these update messages will cause a sustained Denial of Service condition. This issue affects Juniper Networks: Junos OS: All versions prior to 17.3R3-S10 with the exceptions of 15.1X49-D240 on SRX Series and 15.1R7-S8 on EX Series; 17.3 versions prior to 17.3R3-S10; 17.4 versions prior to 17.4R2-S12, 17.4R3-S4; 18.1 versions prior to 18.1R3-S12; 18.2 versions prior to 18.2R2-S8, 18.2R3-S6; 18.3 versions prior to 18.3R3-S4; 18.4 versions prior to 18.4R1-S8, 18.4R2-S6, 18.4R3-S6; 19.1 versions prior to 19.1R1-S6, 19.1R2-S2, 19.1R3-S3; 19.2 versions prior to 19.2R3-S1; 19.3 versions prior to 19.3R2-S5, 19.3R3-S1; 19.4 versions prior to 19.4R1-S3, 19.4R2-S3, 19.4R3; 20.1 versions prior to 20.1R2; 20.2 versions prior to 20.2R1-S3 20.2R2; 20.3 versions prior to 20.3R1-S1, 20.3R2. Junos OS Evolved: All versions prior to 20.3R1-S1-EVO, 20.3R2-EVO."
      }
    ],
    "references": [
      {
        "url": "https://kb.juniper.net/JSA11101",
        "name": "https://kb.juniper.net/JSA11101",
        "refsource": "CONFIRM",
        "tags": [
          "Vendor Advisory"
        ]
      }
    ]
  },
  {
    "cveID": "CVE-2017-12626",
    "cveASSIGNER": "security@apache.org",
    "publishedDate": "2018-01-29T17:29Z",
    "lastModifiedDate": "2021-01-20T15:15Z",
    "description": [
      {
        "lang": "en",
        "value": "Apache POI in versions prior to release 3.17 are vulnerable to Denial of Service Attacks: 1) Infinite Loops while parsing crafted WMF, EMF, MSG and macros (POI bugs 61338 and 61294), and 2) Out of Memory Exceptions while parsing crafted DOC, PPT and XLS (POI bugs 52372 and 61295)."
      }
    ],
    "references": [
      {
        "url": "https://lists.apache.org/thread.html/453d9af5dbabaccd9afb58d27279a9dbfe8e35f4e5ea1645ddd6960b@%3Cdev.poi.apache.org%3E",
        "name": "[dev] 20180126 CVE-2017-12626 - Denial of Service Vulnerabilities in Apache POI < 3.17",
        "refsource": "MLIST",
        "tags": [
          "Mailing List",
          "Vendor Advisory"
        ]
      },
      {
        "url": "http://www.securityfocus.com/bid/102879",
        "name": "102879",
        "refsource": "BID",
        "tags": [
          "Third Party Advisory",
          "VDB Entry"
        ]
      },
      {
        "url": "https://access.redhat.com/errata/RHSA-2018:1322",
        "name": "RHSA-2018:1322",
        "refsource": "REDHAT",
        "tags": [
          "Third Party Advisory"
        ]
      },
      {
        "url": "https://www.oracle.com/technetwork/security-advisory/cpuoct2019-5072832.html",
        "name": "https://www.oracle.com/technetwork/security-advisory/cpuoct2019-5072832.html",
        "refsource": "MISC",
        "tags": []
      },
      {
        "url": "https://lists.apache.org/thread.html/708d94141126eac03011144a971a6411fcac16d9c248d1d535a39451@%3Csolr-user.lucene.apache.org%3E",
        "name": "[lucene-solr-user] 20190104 Re: SOLR v7 Security Issues Caused Denial of Use - Sonatype Application Composition Report",
        "refsource": "MLIST",
        "tags": []
      },
      {
        "url": "https://www.oracle.com/security-alerts/cpujan2020.html",
        "name": "https://www.oracle.com/security-alerts/cpujan2020.html",
        "refsource": "MISC",
        "tags": []
      },
      {
        "url": "https://www.oracle.com/security-alerts/cpuapr2020.html",
        "name": "N/A",
        "refsource": "N/A",
        "tags": []
      },
      {
        "url": "https://www.oracle.com/security-alerts/cpujul2020.html",
        "name": "https://www.oracle.com/security-alerts/cpujul2020.html",
        "refsource": "MISC",
        "tags": []
      },
      {
        "url": "https://www.oracle.com/security-alerts/cpuoct2020.html",
        "name": "https://www.oracle.com/security-alerts/cpuoct2020.html",
        "refsource": "MISC",
        "tags": []
      },
      {
        "url": "https://www.oracle.com/security-alerts/cpujan2021.html",
        "name": "https://www.oracle.com/security-alerts/cpujan2021.html",
        "refsource": "MISC",
        "tags": []
      }
    ]
  },
  {
    "cveID": "CVE-2020-9138",
    "cveASSIGNER": "psirt@huawei.com",
    "publishedDate": "2021-01-13T22:15Z",
    "lastModifiedDate": "2021-01-19T15:03Z",
    "description": [
      {
        "lang": "en",
        "value": "There is a heap-based buffer overflow vulnerability in some Huawei Smartphone, Successful exploit of this vulnerability can cause process exceptions during updating."
      }
    ],
    "references": [
      {
        "url": "https://consumer.huawei.com/en/support/bulletin/2020/12/",
        "name": "https://consumer.huawei.com/en/support/bulletin/2020/12/",
        "refsource": "MISC",
        "tags": [
          "Vendor Advisory"
        ]
      }
    ]
  },
  {
    "cveID": "CVE-2020-8264",
    "cveASSIGNER": "cve-assignments@hackerone.com",
    "publishedDate": "2021-01-06T21:15Z",
    "lastModifiedDate": "2021-01-12T14:17Z",
    "description": [
      {
        "lang": "en",
        "value": "In actionpack gem >= 6.0.0, a possible XSS vulnerability exists when an application is running in development mode allowing an attacker to send or embed (in another page) a specially crafted URL which can allow the attacker to execute JavaScript in the context of the local application. This vulnerability is in the Actionable Exceptions middleware."
      }
    ],
    "references": [
      {
        "url": "https://hackerone.com/reports/904059",
        "name": "https://hackerone.com/reports/904059",
        "refsource": "MISC",
        "tags": [
          "Exploit",
          "Patch",
          "Third Party Advisory"
        ]
      },
      {
        "url": "https://groups.google.com/g/rubyonrails-security/c/yQzUVfv42jk/m/oJWw-xhNAQAJ",
        "name": "https://groups.google.com/g/rubyonrails-security/c/yQzUVfv42jk/m/oJWw-xhNAQAJ",
        "refsource": "MISC",
        "tags": [
          "Mailing List",
          "Third Party Advisory"
        ]
      }
    ]
  },
  {
    "cveID": "CVE-2019-20923",
    "cveASSIGNER": "cna@mongodb.com",
    "publishedDate": "2020-11-23T16:15Z",
    "lastModifiedDate": "2020-11-29T21:02Z",
    "description": [
      {
        "lang": "en",
        "value": "A user authorized to perform database queries may trigger denial of service by issuing specially crafted queries, which throw unhandled Javascript exceptions containing types intended to be scoped to the Javascript engine's internals. This issue affects: MongoDB Inc. MongoDB Server v4.0 versions prior to 4.0.7."
      }
    ],
    "references": [
      {
        "url": "https://jira.mongodb.org/browse/SERVER-39481",
        "name": "https://jira.mongodb.org/browse/SERVER-39481",
        "refsource": "CONFIRM",
        "tags": [
          "Issue Tracking",
          "Vendor Advisory"
        ]
      }
    ]
  },
  {
    "cveID": "CVE-2020-26117",
    "cveASSIGNER": "cve@mitre.org",
    "publishedDate": "2020-09-27T04:15Z",
    "lastModifiedDate": "2020-11-06T01:15Z",
    "description": [
      {
        "lang": "en",
        "value": "In rfb/CSecurityTLS.cxx and rfb/CSecurityTLS.java in TigerVNC before 1.11.0, viewers mishandle TLS certificate exceptions. They store the certificates as authorities, meaning that the owner of a certificate could impersonate any server after a client had added an exception."
      }
    ],
    "references": [
      {
        "url": "https://github.com/TigerVNC/tigervnc/commit/20dea801e747318525a5859fe4f37c52b05310cb",
        "name": "https://github.com/TigerVNC/tigervnc/commit/20dea801e747318525a5859fe4f37c52b05310cb",
        "refsource": "MISC",
        "tags": [
          "Patch",
          "Third Party Advisory"
        ]
      },
      {
        "url": "https://github.com/TigerVNC/tigervnc/commit/b30f10c681ec87720cff85d490f67098568a9cba",
        "name": "https://github.com/TigerVNC/tigervnc/commit/b30f10c681ec87720cff85d490f67098568a9cba",
        "refsource": "MISC",
        "tags": [
          "Patch",
          "Third Party Advisory"
        ]
      },
      {
        "url": "https://github.com/TigerVNC/tigervnc/releases/tag/v1.11.0",
        "name": "https://github.com/TigerVNC/tigervnc/releases/tag/v1.11.0",
        "refsource": "MISC",
        "tags": [
          "Release Notes",
          "Third Party Advisory"
        ]
      },
      {
        "url": "https://github.com/TigerVNC/tigervnc/commit/f029745f63ac7d22fb91639b2cb5b3ab56134d6e",
        "name": "https://github.com/TigerVNC/tigervnc/commit/f029745f63ac7d22fb91639b2cb5b3ab56134d6e",
        "refsource": "MISC",
        "tags": [
          "Patch",
          "Third Party Advisory"
        ]
      },
      {
        "url": "https://bugzilla.opensuse.org/show_bug.cgi?id=1176733",
        "name": "https://bugzilla.opensuse.org/show_bug.cgi?id=1176733",
        "refsource": "MISC",
        "tags": [
          "Issue Tracking",
          "Third Party Advisory"
        ]
      },
      {
        "url": "https://github.com/TigerVNC/tigervnc/commit/7399eab79a4365434d26494fa1628ce1eb91562b",
        "name": "https://github.com/TigerVNC/tigervnc/commit/7399eab79a4365434d26494fa1628ce1eb91562b",
        "refsource": "MISC",
        "tags": [
          "Patch",
          "Third Party Advisory"
        ]
      },
      {
        "url": "https://lists.debian.org/debian-lts-announce/2020/10/msg00007.html",
        "name": "[debian-lts-announce] 20201006 [SECURITY] [DLA 2396-1] tigervnc security update",
        "refsource": "MLIST",
        "tags": [
          "Third Party Advisory"
        ]
      },
      {
        "url": "http://lists.opensuse.org/opensuse-security-announce/2020-10/msg00025.html",
        "name": "openSUSE-SU-2020:1666",
        "refsource": "SUSE",
        "tags": [
          "Third Party Advisory"
        ]
      },
      {
        "url": "http://lists.opensuse.org/opensuse-security-announce/2020-11/msg00024.html",
        "name": "openSUSE-SU-2020:1841",
        "refsource": "SUSE",
        "tags": []
      }
    ]
  },
  {
    "cveID": "CVE-2020-4388",
    "cveASSIGNER": "psirt@us.ibm.com",
    "publishedDate": "2020-10-12T14:15Z",
    "lastModifiedDate": "2020-10-14T13:25Z",
    "description": [
      {
        "lang": "en",
        "value": "IBM Cognos Analytics 11.0 and 11.1 could be vulnerable to a denial of service attack by failing to catch exceptions in a servlet also exposing debug information could also be used in future attacks. IBM X-Force ID: 179270."
      }
    ],
    "references": [
      {
        "url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/179270",
        "name": "ibm-cognos-cve20204388-dos (179270)",
        "refsource": "XF",
        "tags": [
          "VDB Entry",
          "Vendor Advisory"
        ]
      },
      {
        "url": "https://www.ibm.com/support/pages/node/6346922",
        "name": "https://www.ibm.com/support/pages/node/6346922",
        "refsource": "CONFIRM",
        "tags": [
          "Patch",
          "Vendor Advisory"
        ]
      }
    ]
  },
  {
    "cveID": "CVE-2011-1248",
    "cveASSIGNER": "secure@microsoft.com",
    "publishedDate": "2011-05-13T17:05Z",
    "lastModifiedDate": "2020-09-28T12:58Z",
    "description": [
      {
        "lang": "en",
        "value": "WINS in Microsoft Windows Server 2003 SP2 and Server 2008 Gold, SP2, R2, and R2 SP1 does not properly handle socket send exceptions, which allows remote attackers to execute arbitrary code or cause a denial of service (memory corruption) via crafted packets, related to unintended stack-frame values and buffer passing, aka \"WINS Service Failed Response Vulnerability.\""
      }
    ],
    "references": [
      {
        "url": "https://oval.cisecurity.org/repository/search/definition/oval%3Aorg.mitre.oval%3Adef%3A12724",
        "name": "oval:org.mitre.oval:def:12724",
        "refsource": "OVAL",
        "tags": []
      },
      {
        "url": "https://docs.microsoft.com/en-us/security-updates/securitybulletins/2011/ms11-035",
        "name": "MS11-035",
        "refsource": "MS",
        "tags": []
      }
    ]
  },
  {
    "cveID": "CVE-2020-13410",
    "cveASSIGNER": "cve@mitre.org",
    "publishedDate": "2020-08-26T15:15Z",
    "lastModifiedDate": "2020-09-04T16:05Z",
    "description": [
      {
        "lang": "en",
        "value": "An issue was discovered in MoscaJS Aedes 0.42.0. lib/write.js does not properly consider exceptions during the writing of an invalid packet to a stream."
      }
    ],
    "references": [
      {
        "url": "https://github.com/moscajs/aedes/pull/493",
        "name": "https://github.com/moscajs/aedes/pull/493",
        "refsource": "MISC",
        "tags": [
          "Patch",
          "Third Party Advisory"
        ]
      },
      {
        "url": "https://payatu.com/advisory/dos-in-aedes-mqtt-broker",
        "name": "https://payatu.com/advisory/dos-in-aedes-mqtt-broker",
        "refsource": "MISC",
        "tags": [
          "Exploit",
          "Third Party Advisory"
        ]
      }
    ]
  },
  {
    "cveID": "CVE-2018-15913",
    "cveASSIGNER": "cve@mitre.org",
    "publishedDate": "2019-06-20T19:15Z",
    "lastModifiedDate": "2020-08-13T13:46Z",
    "description": [
      {
        "lang": "en",
        "value": "An issue was discovered in Cloudera Manager 5.x through 5.15.0. One type of page in Cloudera Manager uses a 'returnUrl' parameter to redirect the user to another page in Cloudera Manager once a wizard is completed. The validity of this parameter was not checked. As a result, the user could be automatically redirected to an attacker's external site or perform a malicious JavaScript function that results in cross-site scripting (XSS). This was fixed by not allowing any value in the returnUrl parameter with patterns such as http://, https://, //, or javascript. The only exceptions to this rule are the SAML Login/Logout URLs, which remain supported since they are explicitly configured and they are not passed via the returnUrl parameter."
      }
    ],
    "references": [
      {
        "url": "https://www.cloudera.com/documentation/other/security-bulletins/topics/Security-Bulletin.html#cloudera_manager",
        "name": "https://www.cloudera.com/documentation/other/security-bulletins/topics/Security-Bulletin.html#cloudera_manager",
        "refsource": "MISC",
        "tags": [
          "Vendor Advisory"
        ]
      },
      {
        "url": "https://www.cloudera.com/documentation/other/security-bulletins/topics/Security-Bulletin.html",
        "name": "https://www.cloudera.com/documentation/other/security-bulletins/topics/Security-Bulletin.html",
        "refsource": "CONFIRM",
        "tags": [
          "Vendor Advisory"
        ]
      }
    ]
  },
  {
    "cveID": "CVE-2020-9227",
    "cveASSIGNER": "psirt@huawei.com",
    "publishedDate": "2020-07-17T23:15Z",
    "lastModifiedDate": "2020-07-22T20:09Z",
    "description": [
      {
        "lang": "en",
        "value": "Huawei Smart Phones Moana-AL00B with versions earlier than 10.1.0.166 have a missing initialization of resource vulnerability. An attacker tricks the user into installing then running a crafted application. Due to improper initialization of specific parameters, successful exploit of this vulnerability may cause device exceptions."
      }
    ],
    "references": [
      {
        "url": "https://www.huawei.com/en/psirt/security-advisories/huawei-sa-20200715-09-smartphone-en",
        "name": "https://www.huawei.com/en/psirt/security-advisories/huawei-sa-20200715-09-smartphone-en",
        "refsource": "CONFIRM",
        "tags": [
          "Vendor Advisory"
        ]
      }
    ]
  },
  {
    "cveID": "CVE-2018-10872",
    "cveASSIGNER": "secalert@redhat.com",
    "publishedDate": "2018-07-10T19:29Z",
    "lastModifiedDate": "2020-07-15T03:15Z",
    "description": [
      {
        "lang": "en",
        "value": "A flaw was found in the way the Linux kernel handled exceptions delivered after a stack switch operation via Mov SS or Pop SS instructions. During the stack switch operation, processor does not deliver interrupts and exceptions, they are delivered once the first instruction after the stack switch is executed. An unprivileged system user could use this flaw to crash the system kernel resulting in DoS. This CVE-2018-10872 was assigned due to regression of CVE-2018-8897 in Red Hat Enterprise Linux 6.10 GA kernel. No other versions are affected by this CVE."
      }
    ],
    "references": [
      {
        "url": "https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2018-10872",
        "name": "https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2018-10872",
        "refsource": "CONFIRM",
        "tags": [
          "Issue Tracking",
          "Patch",
          "Third Party Advisory"
        ]
      },
      {
        "url": "https://access.redhat.com/errata/RHSA-2018:2164",
        "name": "RHSA-2018:2164",
        "refsource": "REDHAT",
        "tags": [
          "Third Party Advisory"
        ]
      },
      {
        "url": "https://help.ecostruxureit.com/display/public/UADCE725/Security+fixes+in+StruxureWare+Data+Center+Expert+v7.6.0",
        "name": "https://help.ecostruxureit.com/display/public/UADCE725/Security+fixes+in+StruxureWare+Data+Center+Expert+v7.6.0",
        "refsource": "CONFIRM",
        "tags": []
      },
      {
        "url": "https://www.oracle.com/security-alerts/cpujul2020.html",
        "name": "https://www.oracle.com/security-alerts/cpujul2020.html",
        "refsource": "MISC",
        "tags": []
      }
    ]
  },
  {
    "cveID": "CVE-2020-5303",
    "cveASSIGNER": "security-advisories@github.com",
    "publishedDate": "2020-04-10T19:15Z",
    "lastModifiedDate": "2020-06-30T18:15Z",
    "description": [
      {
        "lang": "en",
        "value": "Tendermint before versions 0.33.3, 0.32.10, and 0.31.12 has a denial-of-service vulnerability. Tendermint does not limit the number of P2P connection requests. For each p2p connection, it allocates XXX bytes. Even though this memory is garbage collected once the connection is terminated (due to duplicate IP or reaching a maximum number of inbound peers), temporary memory spikes can lead to OOM (Out-Of-Memory) exceptions. Additionally, Tendermint does not reclaim activeID of a peer after it's removed in Mempool reactor. This does not happen all the time. It only happens when a connection fails (for any reason) before the Peer is created and added to all reactors. RemovePeer is therefore called before AddPeer, which leads to always growing memory (activeIDs map). The activeIDs map has a maximum size of 65535 and the node will panic if this map reaches the maximum. An attacker can create a lot of connection attempts (exploiting above denial of service), which ultimately will lead to the node panicking. These issues are patched in Tendermint 0.33.3 and 0.32.10."
      }
    ],
    "references": [
      {
        "url": "https://github.com/tendermint/tendermint/commit/e2d6859afd7dba4cf97c7f7d412e7d8fc908d1cd",
        "name": "https://github.com/tendermint/tendermint/commit/e2d6859afd7dba4cf97c7f7d412e7d8fc908d1cd",
        "refsource": "MISC",
        "tags": [
          "Patch",
          "Third Party Advisory"
        ]
      },
      {
        "url": "https://github.com/tendermint/tendermint/security/advisories/GHSA-v24h-pjjv-mcp6",
        "name": "https://github.com/tendermint/tendermint/security/advisories/GHSA-v24h-pjjv-mcp6",
        "refsource": "CONFIRM",
        "tags": [
          "Third Party Advisory"
        ]
      },
      {
        "url": "https://hackerone.com/reports/820317",
        "name": "https://hackerone.com/reports/820317",
        "refsource": "MISC",
        "tags": [
          "Permissions Required",
          "Third Party Advisory"
        ]
      }
    ]
  },
  {
    "cveID": "CVE-2017-18207",
    "cveASSIGNER": "cve@mitre.org",
    "publishedDate": "2018-03-01T05:29Z",
    "lastModifiedDate": "2020-01-21T22:15Z",
    "description": [
      {
        "lang": "en",
        "value": "** DISPUTED ** The Wave_read._read_fmt_chunk function in Lib/wave.py in Python through 3.6.4 does not ensure a nonzero channel value, which allows attackers to cause a denial of service (divide-by-zero and exception) via a crafted wav format audio file. NOTE: the vendor disputes this issue because Python applications \"need to be prepared to handle a wide variety of exceptions.\""
      }
    ],
    "references": [
      {
        "url": "https://bugs.python.org/issue32056",
        "name": "https://bugs.python.org/issue32056",
        "refsource": "MISC",
        "tags": [
          "Issue Tracking",
          "Vendor Advisory"
        ]
      },
      {
        "url": "http://lists.opensuse.org/opensuse-security-announce/2020-01/msg00040.html",
        "name": "openSUSE-SU-2020:0086",
        "refsource": "SUSE",
        "tags": []
      }
    ]
  },
  {
    "cveID": "CVE-2019-14853",
    "cveASSIGNER": "secalert@redhat.com",
    "publishedDate": "2019-11-26T13:15Z",
    "lastModifiedDate": "2019-12-17T23:15Z",
    "description": [
      {
        "lang": "en",
        "value": "An error-handling flaw was found in python-ecdsa before version 0.13.3. During signature decoding, malformed DER signatures could raise unexpected exceptions (or no exceptions at all), which could lead to a denial of service."
      }
    ],
    "references": [
      {
        "url": "https://github.com/warner/python-ecdsa/releases/tag/python-ecdsa-0.13.3",
        "name": "https://github.com/warner/python-ecdsa/releases/tag/python-ecdsa-0.13.3",
        "refsource": "MISC",
        "tags": [
          "Release Notes"
        ]
      },
      {
        "url": "https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2019-14853",
        "name": "https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2019-14853",
        "refsource": "CONFIRM",
        "tags": [
          "Issue Tracking",
          "Third Party Advisory"
        ]
      },
      {
        "url": "https://www.debian.org/security/2019/dsa-4588",
        "name": "DSA-4588",
        "refsource": "DEBIAN",
        "tags": []
      },
      {
        "url": "https://seclists.org/bugtraq/2019/Dec/33",
        "name": "20191218 [SECURITY] [DSA 4588-1] python-ecdsa security update",
        "refsource": "BUGTRAQ",
        "tags": []
      }
    ]
  },
  {
    "cveID": "CVE-2019-16768",
    "cveASSIGNER": "security-advisories@github.com",
    "publishedDate": "2019-12-05T20:15Z",
    "lastModifiedDate": "2019-12-17T20:54Z",
    "description": [
      {
        "lang": "en",
        "value": "In affected versions of Sylius, exception messages from internal exceptions (like database exception) are wrapped by \\Symfony\\Component\\Security\\Core\\Exception\\AuthenticationServiceException and propagated through the system to UI. Therefore, some internal system information may leak and be visible to the customer. A validation message with the exception details will be presented to the user when one will try to log into the shop. This has been patched in versions 1.3.14, 1.4.10, 1.5.7, and 1.6.3."
      }
    ],
    "references": [
      {
        "url": "https://github.com/Sylius/Sylius/commit/be245302dfc594d8690fe50dd47631d186aa945f",
        "name": "https://github.com/Sylius/Sylius/commit/be245302dfc594d8690fe50dd47631d186aa945f",
        "refsource": "MISC",
        "tags": [
          "Release Notes"
        ]
      },
      {
        "url": "https://github.com/Sylius/Sylius/security/advisories/GHSA-3r8j-pmch-5j2h",
        "name": "https://github.com/Sylius/Sylius/security/advisories/GHSA-3r8j-pmch-5j2h",
        "refsource": "CONFIRM",
        "tags": [
          "Mitigation",
          "Third Party Advisory"
        ]
      }
    ]
  }
]